doc/config.md

```diff
@@ -2,128 +2,209 @@
+  [DesiredCapabilites](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)
+  for the browser. Per-browser value will get merged with top-level value
+  instead of overriding it.
 
-  ```yaml
-  coverageExclude:
-    - libs/**
-    - path/to/some.css
-    -
-  ```
-* `windowSize` (env: `GEMINI_WINDOW_SIZE`) – specify browser window dimensions (i.e. `1600x1200`).
+* `calibrate` - do this browser needs to perform the calibration procedure. This
+  procedure allows to correctly capture the image in case the particular
+  WebDriver implementation captures browser UI along with WebPage. If you are
+  sure this is not the case for your WebDriver, you can disable it by setting
+  this option to `false`.
+* `httpTimeout` - timeout of HTTP requests to WebDriver, msec.
+
+* `screenshotsDir` – directory to save reference screenshots to.
+  `<projectRoot>/gemini/screens` by default.
+
+* `tolerance` - indicates maximum allowed
+  [CIEDE2000](http://en.wikipedia.org/wiki/Color_difference#CIEDE2000)
+  difference between colors. Used only in non-strict mode. By default its 2.3
+  which should be enough for the most cases. Increasing global default is not
+  recommended, prefer changing tolerance for particular suites or states
+  instead.
+
+* `strictComparison` – test will be considered as failed in case of any kind of
+  error. By default, only noticeable differences are treated as test failure.
+
+* `windowSize` – specify browser window dimensions (i.e. `1600x1200`).
+
+* `sessionPerBrowser` - how many WebDriver sessions launch in parallel for this
+  browser. By default is 1. Increase if you want to speed up your tests.
+
+* `suitesPerSession` - maximum amount of test suites to run in each web driver
+  session. After limit is reached, session gets closed and new session gets
+  started. By default is `.inf` (no limit). Set to smaller number if you
+  experiencing stability problems.
+
+## Overriding settings
+
+All options can also be overridden via command-line flags or environment
+variables.  Priorities are the following:
+
+* command-line option has the highest priority. It overrides environment
+  variable and config file value.
+* environment variable has second priority. It overrides config file value.
+* config file value has the lowest priority.
+* if no command-line option, environment variable or config file option
+  specified, default is used.
+
+To override config setting with CLI option, convert full option path to
+`--kebab-case`. For example, if you want to run tests against different root
+URL, call:
+
+```
+gemini test --root-url http://example.com
+```
+
+To save screenshots for IE9 to different location (considering you have browser
+with `ie9` id in a config):
+
+```
+gemini gather --browsers-ie9-screenshots-dir ./ie9-screens
+```
+
+To override setting with environment variable, convert its full path to
+`snake_case` and add `gemini_` prefix. Above examples will like this if you use
```

Comment:
`Above examples can be rewritten to use environment variables instead of CLI options:`